concent_api/core/models.py

```diff
@@ -721,11 +743,14 @@ class DepositClaim(Model):
     created_at = DateTimeField(auto_now_add=True)
     modified_at = DateTimeField(auto_now=True)
 
+    class Meta:
+        unique_together = ('subtask_id', 'concent_use_case', 'payee_ethereum_address')
+
     def clean(self) -> None:
         super().clean()
-        if self.subtask is None and self.concent_use_case != ConcentUseCase.FORCED_PAYMENT:
+        if self.subtask_id is None and self.concent_use_case != ConcentUseCase.FORCED_PAYMENT:
```

Comment:
Can the ID be an empty string here?

Reply:
No, it is passed directly from `task_to_compute` in our code when `ConcentUseCase` is not `FORCED_PAYMENT`, so it can't happen. And we validate against that earlier.

Reply:
I don't know what do you mean by "everywhere". Basically `subtask_id` comes from `task_to_compute` and must be always uuid (we validate against that). So I think the only place where it is accepted to be something else occasionally is this method.

Reply:
> I don't know what do you mean by "everywhere".

I mean all the Concent code, not just Bankster.